frontend/cypress/integration/landing.spec.js

```diff
@@ -17,6 +17,8 @@ describe('Landing Page Test', () => {
     cy.wait('@statistics');
     cy.get('[data-test-id="statistic-Universités partenaires"]').should('exist');
     cy.get('[data-test-id="statistic-Universités partenaires-value"]').should('have.text', 11);
+    cy.get('[data-test-id="statistic-Nombre de séances réalisées"]').should('exist');
+    cy.get('[data-test-id="statistic-Nombre de séances réalisées-value"]').should('have.text', 156);
```

Comment:
```suggestion
    cy.get('[data-test-id="statistic-Séances réalisées"]').should('exist');
    cy.get('[data-test-id="statistic-Séances réalisées-value"]').should('have.text', 156);
```
(if previous suggestion was applied)